tests/e2e/specs/admin/reader-theme-carousel.js

```diff
@@ -6,6 +6,7 @@ import { visitAdminPage } from '@wordpress/e2e-test-utils';
 describe( 'AMP settings screen reader themes carousel', () => {
 	beforeEach( async () => {
 		await visitAdminPage( 'admin.php', 'page=amp-options' );
+		await page.addStyleTag( { content: 'html {scroll-behavior: auto !important;}' } );
```

Comment:
Figuring this out was rough. Puppeteer kept failing with "Node is either not visible or not an HTMLElement", and I found this buried as the last answer in one of the lower-ranking StackOverflow results. Essentially, the `scroll-behavior: smooth` CSS rule I added to the setting screen messes up Puppeteer.

Reply:
Good find.